clouddriver-dcos/src/test/groovy/com/netflix/spinnaker/clouddriver/dcos/deploy/util/id/MarathonPathIdSpec.groovy

```diff
@@ -0,0 +1,97 @@
+package com.netflix.spinnaker.clouddriver.dcos.deploy.util.id
+
+import spock.lang.Specification
+
+class MarathonPathIdSpec extends Specification {
+    static final def ACCOUNT = "spinnaker"
+
+    void "constructor should throw an IllegalArgumentException if path is invalid"() {
+        expect:
```

Comment:
Spock has a `thrown` keyword that you can use when you expect exceptions to be thrown.